tasty-query/shared/src/main/scala/tastyquery/TypeMaps.scala

```diff
@@ -301,12 +294,12 @@ private[tastyquery] object TypeMaps {
               case Some(sym: ClassTypeParamSymbol) => expandParam(sym, preHi)
               case _                               => tryWiden(tp, preHi)
             forwarded.getOrElse {
-              range(super.derivedSelect(tp, preLo).lowerBound, super.derivedSelect(tp, preHi).upperBound)
+              range(super.derivedSelect(tp, preLo) /*.lowerBound*/, super.derivedSelect(tp, preHi) /*.upperBound*/ )
             }
           case _ =>
             super.derivedSelect(tp, pre) match {
-              case tp: WildcardTypeBounds => range(tp.bounds.low, tp.bounds.high)
-              case tp                     => tp
+              //case tp: WildcardTypeBounds => range(tp.bounds.low, tp.bounds.high)
```

Comment:
is this now illegal or temporary removal?

Reply:
Same here.